app/models/openscap_result.rb

```diff
@@ -0,0 +1,77 @@
+class OpenscapResult < ApplicationRecord
+  belongs_to :container_image
+  has_one    :binary_blob,           :dependent => :destroy, :autosave => true, :as => :resource
+  has_many   :openscap_rule_results, :dependent => :destroy, :autosave => true
+
+  def self.openscap_available?
+    # needed only for travis
+    require 'openscap'
+    require 'openscap/ds/arf'
+    require 'openscap/xccdf/benchmark'
+    true
+  rescue LoadError
+    false
+  end
+
+  def html
+    raw = binary_blob.try(:binary)
+    return unless raw
```

Comment:
I Assume that if the (image has an openscap_result)[https://github.com/ManageIQ/manageiq/pull/6839/files#diff-78122ac21511d34c71d004ea16f3faacR123] it must have a valid html. I've changed the code to reflect that.